### PR TITLE
789 podcast

### LINKED ISSUE #789: podcast

### PR DESCRIPTION
**Pull request by Arfah**
#closes #789 This is a new landing page for the new podcast advert that is being made about remote. The focus of this landing page is about capturing emails.

### CODE REVIEW

File: source/podcast.html.haml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+- description "Makers Academy is London's leading Web Developer Bootcamp. Learn to code in 12 Weeks by enrolling on one of our programming / web development courses! Learn Ruby, JavaScript and the best software practices."
+%section.hero-splash{style: "background-image: url(#{image_path("backgrounds/dan-and-students-overhead-70.jpg")});"}
+  .container
+    .centered-row
+      %h2  Learn Ruby in 15 Minutes
```
Comment (nikeshashar) [inline source/podcast.html.haml]: - [ ] Shouldn't this be a H1
- [ ] Spacing between the %h tag and the start of "Learn" is double

File: source/podcast.html.haml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      %h4.subheader Download the guide
+      %p  Makers Academy is Europe's leading Web Developer bootcamp. With our help, you'll learn the principles of software craftsmanship and we'll also help you get your first job as a developer.
+      %p Sign up now and receive your free learn Ruby in 15 minutes ebook!
```
Comment (nikeshashar) [inline source/podcast.html.haml]: This reads like they are signing up to Makers Academy for a book.

File: source/podcast/success.html.haml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+=javascript_include_tag "track-form/submit-to-analytics"
+:javascript
+  function submitFormDataToAnalytics (formData) {
+    analytics.alias(formData.email);
+    analytics.identify(formData.email, formData);
+    analytics.track("Submitted Digital Gurus form");
```
Comment (nikeshashar) [inline source/podcast/success.html.haml]: - [ ] Analytics says Digital Gurus form.

File: source/podcast/success.html.haml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  }
+%section.hero-splash{style: "background-image: url(#{image_path("backgrounds/riz-and-tom.jpg")});"}
+  .container
+    .centered-row
+      .logo
+        %a{href: "http://www.makersacademy.com"}
+          %img{src: image_path("logo/ma-white.png"), class: "logo"}
+      %h1 Thank you!
+      %p Check your email for the ebook! If you haven't received anything then email <a href="mailto:arfah@makersacademy.com">arfah@makersacademy.com</a>
```
Comment (nikeshashar) [inline source/podcast/success.html.haml]: - [ ] I'd remove your personal email address and put marketing@

File: data/graduates.yml [hunk 5]
```diff
@@ -30,6 +30,32 @@
...
+  summary: Assistant Lecturer to Junior Developer at Sage UK
+  testimonial: An amazing transformative experience that has taught me how to be a developer
+  cohort_month: Remote November
+  cohort_year: 2016
+  categories:
+    - podcast
+    - career-changer
+  case_study:
+    - question: What were you doing before joining Makers Academy?
+      answer: I was working as an Assistant Lecturer in Mathematics.
```
Comment (nikeshashar) [inline data/graduates.yml]: mathematics (lowercase M)

File: data/graduates.yml [hunk 6]
```diff
@@ -30,6 +30,32 @@
...
+    - question: How did you find learning remotley?
```
Comment (nikeshashar) [inline data/graduates.yml]: remotley = typo

File: data/graduates.yml [hunk 7]
```diff
@@ -30,6 +30,32 @@
...
+    - question: How has your life changed since graduation?
+      answer: I have a new career! I am confident I can learn new skills quickly. I know how to approach learning a new skill.
+    - question: What advice would you give to people thinking of changing career?
+      answer: If this a field you want to be in then make the move. A career change does not invalidate all the hard work in the previous career because you bring those skills along. That past experience frames how you approach problems as a developer and will make you unique, and that uniqueness will be just what some person is looking for in the next employee or investment.
+    - question: What would you say to people who say £4000 is expensive for an online course?
+      answer: It is a big investment, but an investment in yourself. MA provides a guided path to becoming a developer. The course content alone makes the course worth it, but MA trains you in industry relevant best practices that will make you appealing to employers. But the hidden gem at MA is the Careers team. They prepare you to enter the job market and help you frame yourself. The entire MA experience was well worth it all.
```
Comment (nikeshashar) [inline data/graduates.yml]: MA should be Makers to remain consistent with the rest of the site

File: source/podcast.html.haml (hunk 1 above)
Comment (Arfah) [inline source/podcast.html.haml]: Nope needs to be h2 or it becomes too big

File: source/podcast.html.haml (hunk 2 above)
Comment (Arfah) [inline source/podcast.html.haml]: For this landing page you are. It's a podcast advert that links people to give us their details for an e-book

**Comment by Arfah** [comment]
can I have a :shipit: nikesh?

**Comment by nikeshashar** [comment]
:shipit:

**Pull request merged**